src/actions/__tests__/swapParent.ts

```diff
@@ -0,0 +1,61 @@
+import { HOME_TOKEN } from '../../constants'
+import exportContext from '../../selectors/exportContext'
+import setCursorFirstMatch from '../../test-helpers/setCursorFirstMatch'
```

Comment:
I like to import `setCursorFirstMatch` as `setCursor` to make the tests more readable.